### PR TITLE
Update how-to-host-and-run-a-basic-wcf-service.md

### PR DESCRIPTION
**Pull request by tnbuig**
service2.cs should be service1.cs

### CODE REVIEW

**Review by mairaw — APPROVED** [review]
Thank you @tnbuig for correcting the class name. I'll wait until the builds are green to merge the PR.

**Comment by mairaw** [comment]
We're having issues with our CI builds. Given the nature of the change, I'll just merge the PR without validation. Thanks again @tnbuig. Your fix should go live in the next day or so.

**Comment by tnbuig** [comment]
Hi thanks for all the updates.

Make me feel like i my change was to rewrite the framework, when i simply
changed one character :-)

**Pull request merged**
